_layouts/base.html

```diff
@@ -7,6 +7,8 @@
 <html lang="en">
   <head>
     <title>{{ page.title | default: site.title }}</title>
+    <link rel="shortcut icon" href="{{ '/public/favicon.ico' | relative_url }}" />
```

Comment:
Can you remove this from this PR and move them into another as they're unrelated to Docker.